sqlagg/sorting.py

```diff
@@ -1,17 +1,23 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
+
 from sqlalchemy import desc, asc
+from sqlalchemy.sql import column
 
 
 class OrderBy(object):
     def __init__(self, column_name, is_ascending=True):
         self.column_name = column_name
         self.is_ascending = is_ascending
 
+    @property
+    def column(self):
+        return column(self.column_name)
```

Comment:
I think I checked all the places in HQ and we don't do anything crazy like ordering by an expression. Though apparently that's been a hidden feature  in UCR for quite a while. I did not check those reports